css/site.css

```diff
@@ -1066,4 +1064,48 @@ a.nav-link {
 
 .radio input[type=radio] {
   margin-top: 5px;
+}
+
+.navbar-brand:focus {
+  outline: 5px auto Highlight;
+  outline: 5px auto -webkit-focus-ring-color;
+}
+
+.btn:focus {
+  outline: 5px auto Highlight;
+  outline: 5px auto -webkit-focus-ring-color;
+}
+a.sr-only {
+  border: 0;
+  clip: rect(0, 0, 0, 0);
+  height: 1px;
+  overflow: hidden;
+  padding: 0;
+  position: absolute;
+  white-space: nowrap;
+  width: 1px;
+}
+
+a.sr-only-focusable:focus,
+a.sr-only:focus {
+  clip: auto;
```

Comment:
@ostafinskim some of the styles here are duplicated from this file:

https://github.com/FitzwilliamMuseum/tessitura-fitz/blob/master/css/material.min.css#L10944

I believe the following styles can be removed from here:

```
clip: auto;
overflow: visible;
white-space: normal;
```

Can you also please add a comment explaining why this is necessary?

It looks like these styles also specifically related to the `.doc-skip` (skip to main content) link at the top of the doucment. So it would be better to use the `.doc-skip` selector here rather than the more generic `a.sr-only-focusable` and `a.sr-only` selectors, as this is likely going to impact other items on the page (e.g. say there was a hidden link in the footer, it might be confusing for it to appear fixed at the top left hand corner of the page when it receives focus).

Reply:
Solved.